index.js

```diff
@@ -47,6 +47,23 @@ const groupByParents = (array, options) => {
 	}, {});
 };
 
+function isObject(o) {
+	return Object.prototype.toString.call(o) === "[object Object]";
+};
```

Comment:
Could you remove that extra semicolon after function definition?

Reply:
OK